Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -196,45 +197,60 @@
 
   public static void launch(FragmentActivity activity, GameFile gameFile)
   {
-    Intent launcher = new Intent(activity, EmulationActivity.class);
-
-    launcher.putExtra(EXTRA_SELECTED_GAMES, scanForSecondDisc(gameFile));
-    launcher.putExtra(EXTRA_SELECTED_TITLE, gameFile.getTitle());
-    launcher.putExtra(EXTRA_SELECTED_GAMEID, gameFile.getGameId());
-    launcher.putExtra(EXTRA_PLATFORM, gameFile.getPlatform());
-    activity.startActivity(launcher);
-  }
+    if (!sStartingEmulation)
```

Comment:
Can be replaced with:
```java
if (sStartingEmulation)
  return;
```

Reply:
This is void so we're not returning anything. Isn't the current code cleaner than the following?
> ```java
> if (sStartingEmulation)
>   // do nothing;
> else
>   doSomething();
> ```

Reply:
Yes, the current code is cleaner than what you just suggested, but we would prefer what sepalani suggested.

Reply:
Inverted

Reply:
Your change doesn't implement sepalani's suggestion.

Just to be clear, the suggestion is that you put the if-return construct at the beginning of the function, followed by the main code of the function without any `else` statement.

Reply:
Not sure what I was thinking. Went with @sepalani's original suggestions.